### PR TITLE
Add an integration test for forcerelay

### PR DESCRIPTION
**Pull request by ImJeremyHe**
Closes: #XXX

## Description

______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules).
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).
- [ ] Tagged *one* reviewer who will be the one responsible for shepherding this PR.

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: tools/forcerelay-test/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        let cell = cells.objects.get(0).unwrap().clone();
+        let data = cell.output_data.unwrap();
+
+        let reader = ClientReader::new_unchecked(data.as_bytes());
+        ClientReader::verify(data.as_bytes(), true).unwrap();
+        let min_slot = reader.minimal_slot().to_entity();
+        let max_slot = reader.maximal_slot().to_entity();
+        let tip_header_root = reader.tip_valid_header_root().to_entity();
+        assert_eq!(uint64_to_u64(min_slot), 5787808);
+        assert_eq!(uint64_to_u64(max_slot), 5787839);
```
Comment (ImJeremyHe) [inline tools/forcerelay-test/src/lib.rs:141]: Here the max slot is not the last slot of this epoch

File: tools/forcerelay-test/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+            (ckb_run, ckb_miner)
+        };
+
+        let mut running_eth_server = Command::new("python3")
+            .arg("eth_mock/server.py")
+            .spawn()
+            .unwrap();
+
+        // Waiting for the finality of the ckb contracts
```
Comment (ashuralyk) [inline tools/forcerelay-test/src/lib.rs:80]: dont't need to wait for that so long, you can just wait for one block of native ckb mining.

**Comment by ashuralyk** [comment]
I find you record so many JSON files for mocking beacon header, but you can combine them into one JSON file with map format for storing these headers, and when you read them from mock rpc, you just need to read from the BTreeMap.

**Comment by ashuralyk** [comment]
some clippy errors need to solve which are not able to be ignored.

**Review by ashuralyk — APPROVED** [review]

**Pull request merged**